rclpy/test/test_client.py

```diff
@@ -183,6 +197,68 @@ def test_get_service_name_after_remapping(self):
         ]
         TestClient.do_test_service_name(test_service_name_list)
 
+    def test_sync_call(self):
+        def _service(request, response):
+            return response
+        cli = self.node.create_client(GetParameters, 'get/parameters')
+        srv = self.node.create_service(GetParameters, 'get/parameters', _service)
+        try:
+            self.assertTrue(cli.wait_for_service(timeout_sec=20))
+            executor = rclpy.executors.SingleThreadedExecutor(context=self.context)
+            executor.add_node(self.node)
+            executor_thread = threading.Thread(
+                target=TestClient._spin_rclpy_node, args=(self.node, executor))
+            executor_thread.start()
+            result = cli.call(GetParameters.Request(), 5)
+            self.assertTrue(result is not None)
+            executor.shutdown()
+            executor_thread.join()
+        finally:
+            self.node.destroy_client(cli)
+            self.node.destroy_service(srv)
+
+    def test_sync_call_slow(self):
+        def _service(request, response):
+            time.sleep(5)
+            return response
+        cli = self.node.create_client(GetParameters, 'get/parameters')
+        srv = self.node.create_service(GetParameters, 'get/parameters', _service)
+        try:
+            self.assertTrue(cli.wait_for_service(timeout_sec=20))
+            executor = rclpy.executors.SingleThreadedExecutor(context=self.context)
+            executor.add_node(self.node)
+            executor_thread = threading.Thread(
+                target=TestClient._spin_rclpy_node, args=(self.node, executor))
+            executor_thread.start()
+            result = cli.call(GetParameters.Request(), 10)
+            self.assertTrue(result is not None)
+            executor.shutdown()
+            executor_thread.join()
+        finally:
+            self.node.destroy_client(cli)
+            self.node.destroy_service(srv)
+
+    def test_sync_call_timeout(self):
+        def _service(request, response):
+            time.sleep(10)
+            return response
+        cli = self.node.create_client(GetParameters, 'get/parameters')
+        srv = self.node.create_service(GetParameters, 'get/parameters', _service)
+        try:
+            self.assertTrue(cli.wait_for_service(timeout_sec=20))
+            executor = rclpy.executors.SingleThreadedExecutor(context=self.context)
+            executor.add_node(self.node)
+            executor_thread = threading.Thread(
+                target=TestClient._spin_rclpy_node, args=(self.node, executor))
+            executor_thread.start()
+            result = cli.call(GetParameters.Request(), 5)
```

Comment:
`500msec` would be enough for timeout to reduce the test completion time?

```suggestion
            result = cli.call(GetParameters.Request(), 0.5)
```

Reply:
Good suggestion thanks.